framework/include/base/MooseApp.h

```diff
@@ -1168,8 +1168,14 @@ class MooseApp : public ConsoleStreamInterface,
   /// this map when removing relationship managers/ghosting functors
   std::unordered_map<RelationshipManager *, std::shared_ptr<GhostingFunctor>> _undisp_to_disp_rms;
 
+  struct DynamicLibraryInfo
+  {
+    void * _library_handle;
+    std::unordered_set<std::string> _entry_symbols;
```

Comment:
```suggestion
    void * library_handle;
    std::unordered_set<std::string> entry_symbols;
```
For public members I like the libMesh paradigm of not having a leading underscore